### PR TITLE
kubel-scale-replicas command

### PR DESCRIPTION
**Pull request by d1egoaz**
`kubel-scale-replicas` to scale a deployment, replica set, replication
controller, or stateful set.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +71,7 @@ p => port forward pod
 l => log popup
 e => exec popup
 j => jab deployment to force rolling update
+S => scale resource replicas
```
Comment (abrochard) [inline README.md:74]: is it `S` or `s`?

Comment (d1egoaz) [reply README.md:74]: Right, I'll change it to use uppercase s in all code, including transient.

File: README.md [hunk 2]
```diff
@@ -70,6 +71,8 @@ p => port forward pod
 l => log popup
 e => exec popup
 j => jab deployment to force rolling update
+S => scale replicas
```
Comment (abrochard) [inline README.md:74]: dupe?

**Review by abrochard — APPROVED** [review]
Thank you!

**Pull request merged**